modules/FindMagnumIntegration.cmake

```diff
@@ -190,6 +191,14 @@ foreach(_component ${MagnumIntegration_FIND_COMPONENTS})
 
             set(_MAGNUMINTEGRATION_${_COMPONENT}_INCLUDE_PATH_NAMES MotionState.h)
 
+        # ImGui integration library
+        elseif(_component STREQUAL ImGui)
+            find_package(imgui)
```

Comment:
where is the find module for this? is it distributed with imgui itself (as a config, maybe)?